File: python/update_mailboxsettings.py
import http.client

conn = http.client.HTTPSConnection("graph.microsoft.com")

payload = "{ }"

headers = { 'content-type': "application/json" }

conn.request("PATCH", "/api/v1.0/me/mailboxSettings", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: javascript/update_mailboxsettings.js
var data = "{ }";

var xhr = new XMLHttpRequest();
xhr.withCredentials = true;

xhr.addEventListener("readystatechange", function () {
  if (this.readyState === this.DONE) {
    console.log(this.responseText);
  }
});

xhr.open("PATCH", "https://graph.microsoft.com/api/v1.0/me/mailboxSettings");
xhr.setRequestHeader("content-type", "application/json");

xhr.send(data);
